tests/drivers/rtc/rtc_api/testcase.yaml

```diff
@@ -9,3 +9,5 @@ tests:
       - api
     filter: dt_alias_exists("rtc")
     depends_on: rtc
+    integration_platforms:
```

Comment:
native_posix is not an integration platform, this test suite is built to run on any platform that has an RTC, real and emulated like native_posix or qemu [see](https://docs.zephyrproject.org/latest/develop/test/twister.html)

Reply:
The change was removed from the change set.